### DISCUSSION TITLE
Menu Bar Items Not Opening Macbook Air M2

### DISCUSSION THREAD

**Discussion by jjulius50** [jordanbaird/Ice#247] [Q&A]
I just purchased a Macbook Air M2 and installed Ice. When I click on an app in either the hidden section or the always-hidden section. the app does not open. I do not have this issue on my Mac Mini M1. I am using v 0.10.2 of Ice and running macOS Sonoma 14.5 on both Macs. Would someone in the community please help me resolve this? Thanks

**Comment by jordanbaird** [comment]
This was actually brought up by a few other people during beta testing. It seemed like every time I would push out a fix for it, it would cause it to break for someone else. So, at the moment, I'm not entirely sure what a universal fix might be.

**Comment by jjulius50** [comment]
Once again, thank you for the timely reply. That’s really strange, but duplicated here for sure. The app on my Mac Mini works exactly as one would expect.

In any event, I will be happy to test further versions you push out.

Jeff
Jeffrey Julius
***@***.***

Notice: This communication is intended only for the use of the individual or entity to which it is addressed and may contain information that is privileged, confidential or exempt from disclosure. If you have received it in error, please advise the sender by reply email and immediately delete the message and any attachments without copying and/or disclosing the contents. Thank you.

**Reply by jordanbaird** [reply]
Is this still a problem as of the latest release?

**Comment by jjulius50** [comment]
Sorry, I can’t answer that since I removed the app from my work flow.

Jeff
Jeffrey Julius
***@***.***

Notice: This communication is intended only for the use of the individual or entity to which it is addressed and may contain information that is privileged, confidential or exempt from disclosure. If you have received it in error, please advise the sender by reply email and immediately delete the message and any attachments without copying and/or disclosing the contents. Thank you.
